### ISSUE TITLE
Special characters

### ISSUE THREAD

**Issue by Mingo3347** [LNReader/lnreader-plugins#874]
### Source information

فضاء الروايات 

### Steps to reproduce

Open any novel in (فضاء الروايات) source and the chapters contains tons of special characters between every single paragraph making it very annoying........this didn't exist in the previous version of the app but only after I updated to the beta version of 2.0.0
![Screenshot_20240407-043850_LNReader](https://github.com/LNReader/lnreader-plugins/assets/108782331/ffba8d9b-92b9-4433-bc39-a4339e459f98)

### Expected behavior

Only the chapters and the content of the novel should be visible

### Actual behavior

Special characters

### LNReader version

2.0.0

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.19](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Mingo3347** [comment]
Will the problem be fixed in the next update? Because I tried the source after receiving that the issue is closed in the 2.0 version and now it's just an infinite loading screen whenever I click on a novel.Sent from my Galaxy
-------- Original message --------From: nyagami ***@***.***> Date: 5/28/24  11:16 AM  (GMT+02:00) To: LNReader/lnreader-plugins ***@***.***> Cc: Mingo3347 ***@***.***>, Author ***@***.***> Subject: Re: [LNReader/lnreader-plugins] Special characters (Issue #874) 
Closed #874 as completed via #920.

**Comment by nyagami** [comment]
ye, I think it is broken again, should be another issud

**Comment by Mingo3347** [comment]
Issue submitted.Sent from my Galaxy
-------- Original message --------From: nyagami ***@***.***> Date: 5/30/24  5:14 AM  (GMT+02:00) To: LNReader/lnreader-plugins ***@***.***> Cc: Mingo3347 ***@***.***>, Author ***@***.***> Subject: Re: [LNReader/lnreader-plugins] Special characters (Issue #874) 
ye, I think it is broken again, should be another issud
